core/src/css/component/duels/desktop/filters/_duels-filters.component.scss

```diff
@@ -31,7 +31,6 @@
 .hide-below-threshold-link,
 .show-hidden-decks-link {
 	align-self: flex-end;
-	flex-grow: 1;
```

Comment:
Could you also send a screenshot of how it looks like when the window has its minimum size?

Reply:
![2022-09-13_16-39-42](https://user-images.githubusercontent.com/43519401/189916468-90e2a792-fbb3-4d43-b3c4-f5faf0641896.png)

Reply:
Perfect, thanks